src/crimson/osd/watch.h

```diff
@@ -65,9 +67,7 @@ class Watch : public std::enable_shared_from_this<Watch> {
     // NOP
   }
 
-  seastar::future<> remove(bool) {
-    return seastar::now();
-  }
+  seastar::future<> remove(bool);
```

Comment:
nit, i think it'd be better to name the parameter here, unless the type is self-documented, as it's a part of the public interface and document.

Reply:
True. Introduced the name.